log/sequencer.go

```diff
@@ -136,14 +136,12 @@ func (s Sequencer) buildMerkleTreeFromStorageAtRoot(ctx context.Context, root *t
 	mt, err := compact.NewTreeWithState(s.hasher, int64(root.TreeSize), func(depth int, index int64) ([]byte, error) {
 		nodeID, err := storage.NewNodeIDForTreeCoords(int64(depth), index, maxTreeDepth)
 		if err != nil {
-			glog.Warningf("%x: Failed to create nodeID: %v", s.signer.KeyHint, err)
-			return nil, err
+			return nil, fmt.Errorf("%x: Failed to create nodeID: %v", s.signer.KeyHint, err)
```

Comment:
This is fairly serious. We might still want to log it but not necessary if it caller definitely logs.

Reply:
It ends up getting logged at ERROR level here: https://github.com/google/trillian/blob/2609dcb936ec037623152c28146867fffdc07276/server/log_operation_manager.go#L414